### ISSUE TITLE
Talk Proposal: Accessibility, Inputs & JavaScript

### ISSUE THREAD

**Issue by ovlb** [berlinjs/berlinjs.org#290]
##### Talk Title

Accessbility, Inputs & JavaScript

##### Abstract

At the end of February, WebAIM published The WebAIM Million, an accessibility analysis of the homepages of the one million most visited websites.

The findings are depressing. The average site contains about 60 errors. Things went even more downhill when putting JS frameworks in the equation. As a reaction to those findings, this talk tries to shed light on some accessibility best practices regarding accessible inputs, as well as do soulsearching why we, as developers, have failed so badly.

---

Disclaimer: I have given a version of this talk at Vue.js Berlin. Slides can be found here: https://noti.st/ovl/OY4mvA/accessibility-inputs-vue

---

###### Your name

Oscar 

###### Means to get in touch with you

o [at] ovl.design
[_ovlb](https://twitter.com/_ovlb)

**Comment by carolstran** [comment]
👋 @ovlb, thanks for submitting! This topic is very near and dear to my heart. I've put you on the waitlist.

Quick question for you though: Is there any chance that you'd be open to delivering a 5-10 minute version of your talk?

If not, no worries - we can keep you on the regular waitlist. Just trying to think of ways to get more people on our stage before the end of 2019  🤔💡 ⚡️ 🚀

**Comment by ovlb** [comment]
Hello, Carolyn 👋 

For this talk I would need some more time to make the point for inclusive design/development. As there is such a profound number of errors I think it’s important to deliver some background. 

What I could do, deviating from the original concept, is something in this direction, without the focus on a specific framework. 
https://www.matuzo.at/blog/12-tips-for-more-accessible-react-apps-slides-react-finland-2019/

**Comment by carolstran** [comment]
Hey hey @ovlb, no problem we can keep it the full-length. Fair warning though, we probably won't have a spot for you until early 2020 😞 The dates for Q1 2020 are January 16, February 20 and March 19 (and of course, every third Thursday every month after that) if any of those already look good for you!

**Comment by carolstran** [comment]
👋 Hey @ovlb, would you be available for either our February 20 or March 19 meetups?

**Comment by ovlb** [comment]
Hello, @carolstran! February would fit nicely. :)

**Comment by carolstran** [comment]
Awesome! We'll check in again towards the end of the month 😊

**Comment by ovlb** [comment]
@carolstran Thanks again for having me. Was a pleasure speaking for you :) 

Closing this as the talk has been given.
